src/pudl/metadata/resources/eia861.py

```diff
@@ -75,18 +84,15 @@
         "etl_group": "eia861",
     },
     "core_eia861__yearly_demand_response": {
-        "description": (
-            """Energy demand response programs by state, sector, and balancing
-authority. We collect data for the number of customers enrolled, energy savings,
-potential and actual peak savings, and associated costs.
-
-The EIA861 demand-side management (DSM) table (split into three normalized tables in
+        "description": {
+            "additional_summary_text": "demand response programs by state, sector, and balancing authority.",
+            "additional_details_text": """The EIA861 demand-side management (DSM) table (split into three normalized tables in
```

Comment:
Removed on purpose or accident? (I saw your note in the sheet; is it because this info belongs with the source and not the table?)

```suggestion
            "additional_details_text": """We collect data for the number of customers enrolled, energy savings,
potential and actual peak savings, and associated costs.

The EIA861 demand-side management (DSM) table (split into three normalized tables in
```